IsoCountries.swift

```diff
@@ -25,8 +25,8 @@ class IsoCountries {
     class func flag(countryCode: String) -> String {
         var string = ""
         var country = countryCode.uppercased()
-        for uS in country.unicodeScalars {
-            string.unicodeScalars.append(UnicodeScalar(127397 + uS.value)!)
+        for scalar in country.unicodeScalars {
+            string.unicodeScalars.append(UnicodeScalar(127397 + scalar.value)!)
```

Comment:
Sorry - I just left a comment but it seems to have disappeard :(

My remark was - can you `guard` the appending of the scalar? To make sure it does not crash while for unwrapping.

Also can you explain the `127397` magic number? I have honestly no idea what it is. (is it an offset?)